Austin_Marga_HW8_1.ipynb

```diff
@@ -0,0 +1,1230 @@
+{
```

Comment:
<p>you can do a lot with the format syntax that you use. See: https://docs.python.org/3/library/string.html#format-string-syntax</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ubsuny/CP1_Calculus/pull/3/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='41'/>